packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -221,29 +230,32 @@ export const BasicMultiCombobox: React.FC = () => {
           <Input
             id={uid}
             type="text"
-            {...getInputProps(getDropdownProps({preventKeyAction: isOpen}))}
-            {...getToggleButtonProps({tabIndex: 0})}
-            value={selectedItem || ''}
+            {...toggleButtonProps}
+            {...getInputProps({
+              ref,
+              ...getDropdownProps({preventKeyAction: isOpen}),
+            })}
           />
         </Box>
-        {isOpen && (
-          <ComboboxListbox {...getMenuProps()}>
-            <ComboboxListboxGroup>
-              {filteredItems.map((item, index) => (
+        <ComboboxListbox hidden={!isOpen} {...getMenuProps()}>
```

Comment:
Fixes a downshift console error, downshift expects the `<ul>` to always be in the DOM.